stock_account_valuation_report/views/product_product_views.xml

```diff
@@ -38,6 +39,7 @@
                     name="action_view_amls"
                     type="object"
                     icon="fa-info-circle"
+                    title="View AMLS"
```

Comment:
@RicardCForgeFlow I'd prefer explicit wording (View Journal Items).

Reply:
Okay, thanks for the review :)